### DISCUSSION TITLE
airflow initdb: undefined symbol: Py_GetArgcArgv

### DISCUSSION THREAD

**Discussion by iseessel** [apache/airflow#18393] [Q&A]
**Apache Airflow version**: 1.10.12

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`): N/A

**Environment**:

- **Cloud provider or hardware configuration**: Local Env
- **OS** (e.g. from /etc/os-release): macOS Catalina, Version 10.15.6
- **Kernel** (e.g. `uname -a`): Hybrid (XNU)
- **Install tools**: pip3
- **Python Version**: Python 3.7.3 (natively installed on macOS Catalina).
-**Pip Version**: pip 20.2.3

**What happened**:

<img width="1193" alt="Screen Shot 2020-09-15 at 4 18 00 PM" src="https://user-images.githubusercontent.com/25669348/93260197-0b39f680-f76f-11ea-8ade-708a8303a3dd.png">

**What you expected to happen**:

Airflow initdb passes. Airflow 1.10.12 should support Python version 3.7.3. Solution is to upgrade python version, however, my airflow code is not compatible with higher python versions. See https://stackoverflow.com/questions/60684146/airflow-initdb-undefined-symbol-py-getargcargv

**How to reproduce it**:

See above.

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
I am afraid, there is not much we can do other than encouraging setproctitle maintainers to fix it (which I just did). I hope they will resolve it.

**Comment by potiuk** [comment]
@iseessel -> I suggest that you also explain your case in the https://github.com/dvarrazzo/py-setproctitle/issues/82 - the more people will say it is important to them, the better.

**Comment by iseessel** [comment]
@potiuk Was afraid about this -- thanks for the update.
